includes/migrations/class-update-service-data-from-server.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Class Update_Service_Data_From_Server
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Migrations;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class Update_Service_Data_From_Server
+ *
+ * Ensures that the service data is being updated after the upgrade to UPE.
+ * Updating the service data ensures that the payment method fees are present.
+ * If the fees are not present, the plugin shows "missing fees" text next to the UPE payment methods.
+ *
+ * @since 2.8.0
+ */
+class Update_Service_Data_From_Server {
+	/**
+	 * Instance to get information about the account.
+	 *
+	 * @var \WC_Payments_Account
+	 */
+	private $account;
+
+	/**
+	 * Update_Service_Data_From_Server constructor.
+	 *
+	 * @param \WC_Payments_Account $account instance to get information about the account.
+	 */
+	public function __construct( \WC_Payments_Account $account ) {
+		$this->account = $account;
+	}
+
+	/**
+	 * Checks whether it's worth doing the migration.
+	 */
+	public function maybe_migrate() {
+		$account_data = $this->account->get_cached_account_data();
```

Comment:
Would `refresh_account_data()` be a better option in case account data was returning old rates?

Reply:
ok i just see it on line 63, nvm